contracts/TopShotMarketV2.cdc

```diff
@@ -0,0 +1,295 @@
+/*
+
+    TopShotMarketV2.cdc
```

Comment:
It seems there is a naming convention inconsistency between `MarketTopShot.cdc` and `TopShotMarket.cdc` - should we rename this `MarketTopShotV2.cdc`?